builtin/credential/cert/path_certs.go

```diff
@@ -41,9 +42,9 @@ seconds. Defaults to system/backend default TTL.`,
 			},
 
 			"ttl": &framework.FieldSchema{
-				Type: framework.TypeInt,
-				Description: `TTL time in seconds. Defaults to system/backend
-default TTL time.`,
+				Type: framework.TypeString,
```

Comment:
I don't see a `StringDur[ation]`. I do see a TypeDurationSecond, which is an integer.

This change is to bring it in line with how input `ttl`/`lease` parameters are used throughout nearly every other backend, which is a string in `time.ParseDuration`-compatible format. If we want to change that, say to an integer number of seconds, that would be a good candidate for `RoleFramework` to standardize.